website/content/en/docs/User Guide/Explore your project in GCP/_index.md

```diff
@@ -30,9 +30,14 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
 ![image](/docs/images/user-guide/2-hipstershop.png)
 
 ## Run the load generator
+Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to simulate users traffic.  
 
-In another browser tab, navigate to the load-generator URL, from which you can simulate users interacting with the application to generate traffic. For this application, values like 100 total users with a spawn rate of 2 users per second are reasonable. Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't prepopulated. Click the **Start swarming** button to begin generating traffic to the site.
+- In another browser tab, navigate to the load-generator URL(from the installation stage if it isn't populated).  
+- Enter the number of **users** and **spawn rate**. For this application, we recommend to test 100 total users with a spawn rate of 2 users per second.  
+- Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't populated.  
+- Click the **Start swarming** button to begin generating traffic to the site.
 
+This will produce traffic on the store from a loadgenerator pod:
```

Comment:
Isn't it conflicting with your other PR where you make loadgen instruction changes?

Reply:
It's not conflicting, but unneeded. Reverted.